### ISSUE TITLE
Flesh out config values for increased/full customability

### ISSUE THREAD

**Issue by abduct** [thelounge/thelounge#4298]
There are a handful of values which seem to not be able to be set in the config file.

Such values are:
-Socks proxy information in default hosts
-Prefetch user agent
-etc

**Comment by brunnre8** [comment]
Generally TL should stay somewhat simple.
It doesn't have a gazillion config knobs by design.

Still, a catch all issue that regularly grows is a mess, discussions about various barely related things will be interleaved and hard to follow.

Can you please close this catch all and make specific requests when you are sure how you think they should work?

**Comment by brunnre8** [comment]
Let's handle this in individual tickets.
